pkg/core/resources/apis/meshservice/generate/generator.go

```diff
@@ -25,24 +26,27 @@ import (
 )
 
 const (
-	managedByValue string = "meshservice-generator"
+	managedByValue          string = "meshservice-generator"
+	gracePeriodStartedLabel string = "kuma.io/grace-period-started-at"
 )
 
 // Generator generates MeshService objects from Dataplane resources created on
 // universal.
 type Generator struct {
-	logger     logr.Logger
-	interval   time.Duration
-	metric     prometheus.Summary
-	resManager manager.ResourceManager
-	meshCache  *mesh.Cache
+	logger              logr.Logger
+	generateInterval    time.Duration
+	gracePeriodInterval time.Duration
```

Comment:
Is it interval?

Reply:
it is, but it's also a period so the name's definitely redundant